src/_includes/article.html

```diff
@@ -1,30 +1,12 @@
-{% if item.category %}
-  {% if item.asset %}
-    {% assign url = item.asset %}
-    {% unless url contains "https://" %}
-      {% assign url = "/assets/" | append: url %}
-    {% endunless %}
-  {% else %}
-    {% assign url = item.url %}
-  {% endif %}
-{% else %}
-  {% assign url = item.url %}
-  {% if item.external %}
-    {% assign url = item.external %}
-  {% endif %}
-{% endif %}
+{% if item.category %} {% if item.asset %} {% assign url = item.asset %} {% unless url contains "https://" %} {% assign url =
+"/assets/" | append: url %} {% endunless %} {% else %} {% assign url = item.url %} {% endif %} {% else %} {% assign url = item.url
+%} {% if item.external %} {% assign url = item.external %} {% endif %} {% endif %}
```

Comment:
Gonna just add Prettier ignores I think.... this seems to be the Prettier setting's doing.

Reply:
I think it is related to the language mode of the file you have open...

It should be `liquid` for HTML files, and that tells VS Code to use that extension (installed in our devcontainer) to format those files instead of prettier.

But I have seen (e.g. https://github.com/cal-itp/benefits/pull/2270#discussion_r1704658637) sometimes after opening the project in VS Code it will revert the language mode:

![image](https://github.com/user-attachments/assets/52db5d82-50cc-4555-8e88-07827b5921dc)

Reply:
I have it on Liquid, and when I save, Prettier formats it into what the past commit looks like. 

But! I tried pasting in that file into this: https://shopify.github.io/prettier-plugin-liquid/

And it looks like this:

<img width="1337" alt="image" src="https://github.com/user-attachments/assets/0393431d-e8a2-4440-b80e-9ded415c23b6">

Nicely formatted

Reply:
If it reverts to `HTML`, I see the same formatting behavior as you:

![image](https://github.com/user-attachments/assets/7531c293-2dcf-485d-9b9f-5ef8bedcd2aa)

Reply:
This association should be handled by the [`settings.json`](https://github.com/cal-itp/calitp.org/blob/main/.vscode/settings.json#L11) for the devcontainer, but... maybe we need to update those settings?

Reply:
https://github.com/Shopify/theme-tools/tree/main/packages/prettier-plugin-liquid

Reply:
Just to confirm, do you see this extension installed in VS Code?

![image](https://github.com/user-attachments/assets/3cf01079-1bcc-4a51-b67a-904961422053)

Reply:
I think we need to add this to `settings.json` to force that plugin to be the formatter by default:

```diff
"files.associations": {
  "*.html": "liquid"
- }
+ },
+ "[liquid]": {
+   "editor.defaultFormatter": "sissel.shopify-liquid"
+ }
```

Reply:
Yeah I do have that Panoply/Liquid extension installed and activated. And the file is set to Liquid.

Reply:
I hesitate to introduce a new plugin for Prettier which seems like it would:

- Require a JS/npm build pipeline (it's an NPM package)
- Be a plugin of a plugin of VS Code, which seems... confusing?